CMakeLists.txt

```
@@ -15,4 +15,5 @@ set(GZ_SIM_VER ${gz-sim7_VERSION_MAJOR})
 
 add_subdirectory(libraries)
 add_subdirectory(plugins)
+add_subdirectory(tests)
```

I suggest the following modification, based on the docs in https://cmake.org/cmake/help/book/mastering-cmake/chapter/Testing%20With%20CMake%20and%20CTest.html :

```suggestion
option(BUILD_TESTING "Create tests using CMake" OFF)
enable(CTest)
if(BUILD_TESTING)
  add_subdirectory(tests)
endif()
```

Explanation:
* Instead of calling enable_testing directly, the CMake docs suggest to just include the `CTest` module
* The `CTest` module defines the `BUILD_TESTING` variable, and if that variable is set to `OFF`, no tests are enabled. For this reason, if `BUILD_TESTING` is OFF, we do not add the `tests` subdirectory. Furthermore, tipically in our project we set the default value of `BUILD_TESTING` to `OFF`, so that users that do not need to compile the tests do not spend time compiling them.